packages/ember-simple-auth/addon/utils/assign.js

```diff
@@ -1,3 +1,12 @@
-import { assign } from '@ember/polyfills';
+import { importSync } from '@embroider/macros';
+
+let assign;
+
+try {
+  assign = importSync('@ember/polyfills').assign;
+} catch (error) {
+  // Couldn't import @ember/polyfills
+  // Doesn't exist in v5
+}
```

Comment:
It wraps importSync in try catch becasue `dependencySatisfies` macro can't reliably work if the dependency isn't a direct dependency or a peer dependency.
Seems like `ember-source` matching in addons misbehaves especially

Reply:
ember-source needs to be a peerDependency of this addon, I think. That should make it work.

Reply:
Thanks for the pointer :+1: , I also realized this later on but I did get some breakage under fastboot while working on https://github.com/mainmatter/ember-cookies/pull/782